packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1525,6 +1525,24 @@ mixin WidgetInspectorService {
     return false;
   }
 
+  /// Notify debuggers to open an inspector on the object.
+  void _sendInspectEvent(Object? object){
+    inspect(object);
+
+    final _Location? location = WidgetInspectorService.instance._getSelectedSummaryWidgetLocation(null, 'my-group');
+    if (location != null){
+      postEvent(
+        'navigate',
```

Comment:
Should we add some context to this so we know it came from the inspector? For example in future if we add `navigate` events from other sources, it might be useful for clients to have some idea where they came from (for example if the user wanted to disable navigation from some sources)?

For ex:
```
'source': 'inspector',
```

Reply:
While working on the VS Code changes, I noticed we have an option in VS Code to automatically cancel widget-selection mode after you select a widget, but that relies on knowing the navigation was from the inspector - so I do think there's good reason to have some context here.

Reply:
Thanks! :)